internal/handlers/types.go

```diff
@@ -1,6 +1,7 @@
 package handlers
 
 import (
+	"encoding/json"
```

Comment:
I suspect you didn't mean to update the git index of just this file from `0644` to `0755`, if not might be worthwhile reverting it?

Reply:
I've blanket set the file mode to 0644 in most major areas.

Reply:
Oddly I didn't see this file marked as changed.

Reply:
Very odd.